sample.env

```diff
@@ -20,3 +30,16 @@ CHANNELMISMATCH_NOTIFICATION_FROM="Name4 <user4@domain.com>"
 USE_MEMPOOL_FEE_ESTIMATION=true
 MEMPOOL_API_BASE_URL=https://mempool.space/api/v1/
 NODES='[ { "name": "<LSP NAME>", "nodePubkey": "<LIGHTNING NODE PUBKEY>", "lspdPrivateKey": "<LSPD PRIVATE KEY>", "token": "<ACCESS TOKEN>", "host": "<HOSTNAME:PORT for lightning clients>", "publicChannelAmount": "1000183", "channelAmount": "100000", "channelPrivate": false, "targetConf": "6", "minHtlcMsat": "600", "baseFeeMsat": "1000", "feeRate": "0.000001", "timeLockDelta": "144", "channelFeePermyriad": "40", "channelMinimumFeeMsat": "2000000", "additionalChannelCapacity": "100000", "maxInactiveDuration": "3888000", "lnd": { "address": "<HOSTNAME:PORT>", "cert": "<LND_CERT base64>", "macaroon": "<LND_MACAROON hex>" } }, { "name": "<LSP NAME>", "nodePubkey": "<LIGHTNING NODE PUBKEY>", "lspdPrivateKey": "<LSPD PRIVATE KEY>", "token": "<ACCESS TOKEN>", "host": "<HOSTNAME:PORT for lightning clients>", "publicChannelAmount": "1000183", "channelAmount": "100000", "channelPrivate": false, "targetConf": "6", "minHtlcMsat": "600", "baseFeeMsat": "1000", "feeRate": "0.000001", "timeLockDelta": "144", "channelFeePermyriad": "40", "channelMinimumFeeMsat": "2000000", "additionalChannelCapacity": "100000", "maxInactiveDuration": "3888000", "cln": { "pluginAddress": "<address the lsp cln plugin listens on (ip:port)>", "socketPath": "<path to the cln lightning-rpc socket file>" } } ]'
+# If lnd is not used, the "lnd" key can be omitted. The same for applies to cln.
+#
+# TOKEN is a secret shared between the LSP (the lspd instance) and breez-server
+# and is put in the header of each request.
+# You can generate it using for instance the command: openssl rand -base64 24
+#
+# LSPD_PRIVATE_KEY is a key generated and printed in the console when you run
+# ./lspd genkey".
+# When sending the lspd information to the client (the app), lspd adds the
+# public key corresponding to this private key and the client encrypt all the
+# messages sent to the LSP using this public key.
+# The goal is to hide anything from breez-server which is the pass "opaque" data
+# from the app to lspd.
```

Comment:
Maybe for the node configuration values we can add documentation comments in config.go and refer to that here.